packages/dev-frontend/src/pages/Charts.tsx

```diff
@@ -0,0 +1,39 @@
+import { Container, Heading } from "theme-ui";
+
+import { Chart } from "../components/Dashboard/Chart/Chart";
+import { ChartProvider } from "../components/Dashboard/Chart/context/ChartProvider";
+
+export const Charts = (): JSX.Element => (
+    <Container>
+      <Heading as="h2" sx={{ mt: "2.5em", fontWeight: "semibold" }}>
+        Historical Charts
+      </Heading>
+      <Container variant="dashboardGrid">
+        <ChartProvider dataSource='lastGoodPrice'>
```

Comment:
It'd be nice to also include informational hints that explain what these mean
<img width="221" alt="image" src="https://github.com/LiquidEnergyDollar/stability-protocol/assets/11261382/73f470dc-8797-4556-9eeb-fcb0b6c7e4c9">

Reply:
Added tooltips to charts